docs/reference/ml/df-analytics/apis/get-inference-trained-model-stats.asciidoc

```diff
@@ -97,11 +158,25 @@ The API returns the following results:
   "trained_model_stats": [
     {
       "model_id": "flight-delay-prediction-1574775339910",
-      "pipeline_count": 0
+      "pipeline_count": 0,
+      "inference_stats": {
+        "missing_all_fields_count": 0,
+        "inference_count": 4,
+        "cache_miss_count": 3,
+        "failure_count": 0,
```

Comment:
The fields here are in a different order to what `toXContent()` prints.  The docs will tie more closely to what real users see if the fields are in the same order.

Same a few lines below.